package.json

```diff
@@ -5,10 +5,14 @@
   "main": "index.js",
   "scripts": {
     "test": "jest spec",
-    "start": "node db/fake_data.js && nodemon ./server/index.js",
+    "start": "nodemon ./server/index.js",
     "data": "node db/fake_data.js",
     "build": "webpack --mode development --watch",
-    "build-prod": "webpack --mode production"
+    "build-prod": "webpack --mode production",
+    "seedp": "psql postgres < postgresSchema.sql",
+    "genp": "node --max-old-space-size=16384 db/seedCsvs.js",
+    "seedm": "mongoimport -d bookings_portals -c listingBookings --type csv --file db/listingBookings_data.json --headerline",
+    "genm": "node --max-old-space-size=16384 db/seedNonRelationalFile.js"
```

Comment:
Always great to add package.json scripts for commonly used commands. Looks great.